src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl {
+    public void write(String content, String fileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            writer.write(content);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Consider using a logging framework instead of `printStackTrace()` for better error handling. This approach will help manage and track errors more effectively, especially in a production environment.